include/NAS2D/MathUtils.h

```diff
@@ -29,5 +30,26 @@ namespace NAS2D
 
 	int divideUp(int to_divide, int divisor);
 
+	
+
+/**
+ * \fn T convertDomainToRange(T a, T b)
```

Comment:
The name `convertDomainToRange` is pretty generic, and could be descriptive of *any* function.

I would recommend `scaleLinear`, or similar. I believe the "linear" aspect is important, and should be part of the name. I might also be willing to entertain something like `mapLinear`. Not to be confused with [map](https://en.wikipedia.org/wiki/Map_(higher-order_function)) found in several popular programming languages.